### PR TITLE
adding navigation and folder architecture

### PR DESCRIPTION
**Pull request by JorgeGarciaIrazabal**
-adding navigation and folder architecture.

-created a mock of the login page

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+const UIManager = NativeModules.UIManager;
+
+export default class App extends Component {
+  static configureScene(route) {
+    return route.animationType || Navigator.SceneConfigs.FloatFromRight;
+  }
+  static renderScene(route, navigator) {
+    return (
+      <View style={core_styles.container}>
```
Comment (JorgeGarciaIrazabal) [inline src/app.js:16]: Base view will have the status bar and the toolbar

File: src/pages/login.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import {View, StyleSheet} from 'react-native';
+import React, {Component, PropTypes} from 'react';
+import {routes} from '../routes';
+import core_styles from '../styles/core-styles';
+import FontAwesomeIcon from 'react-native-vector-icons/FontAwesome';
+import {Sae} from 'react-native-textinput-effects';
```
Comment (Fdom92) [inline src/pages/login.js:6]: Awesome input effects!! dont know about this.. great! 🎉 🎉

File: index.android.js [hunk 3]
```diff
@@ -1,53 +1,14 @@
...
-    margin: 10,
-  },
-  instructions: {
-    textAlign: 'center',
-    color: '#333333',
-    marginBottom: 5,
-  },
-});
-
 AppRegistry.registerComponent('React_Native_Chat', () => React_Native_Chat);
```
Comment (Fdom92) [inline index.android.js:14]: Maybe the same for iOS? 😃

File: package.json [hunk 4]
```diff
@@ -1,27 +1,29 @@
...
+    "start": "node node_modules/react-native/local-cli/cli.js start",
+    "test": "jest"
+  },
+  "dependencies": {
+    "mobx": "^3.1.0",
+    "mobx-react": "^4.1.0",
+    "react": "~15.4.0",
+    "react-native": "0.41.2",
+    "react-native-drawer": "^2.3.0",
+    "react-native-material-ui": "^1.7.0",
```
Comment (Fdom92) [inline package.json:15]: material design! 😍 great

**Comment by Fdom92** [comment]
@JorgeGarciaIrazabal I installed this on my android device and works pretty good! And because we don't have anything defined about navigation or pages you can merge this without problem. 

On the future we will define navigation and pages and will have to add more into routes file.

File: index.android.js (hunk 3 above)
Comment (JorgeGarciaIrazabal) [reply index.android.js:14]: Who cares about iOS ? 😉

Comment (Fdom92) [reply index.android.js:14]: 😮 xD

**Pull request merged**